web/src/utils/units.ts

```diff
@@ -0,0 +1,23 @@
+export enum PowerUnits {
+  WATTS = 'W',
+  KILOWATTS = 'kW',
+  MEGAWATTS = 'MW',
+  GIGAWATTS = 'GW',
+  TERAWATTS = 'TW',
+  PETAWATTS = 'PW',
```

Comment:
Peta watts is probably not needed but I wanted parity with the energy units (where it might be needed).